.github/workflows/ci_docs.yml

```diff
@@ -113,7 +113,7 @@ jobs:
             sphinx-autodoc-typehints
             sphinx-copybutton
             sphinx-design
-            sphinx-gallery<0.17.0
+            sphinx-gallery
```

Comment:
I am wondering if we should have two PRs:
* one for moving the projection table to fix issue #3346 
* one for unpinning the sphinx-gallery version to revert PR #3350

Reply:
I think one PR is OK